main_test.go

```diff
@@ -0,0 +1,336 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/SkynetLabs/skynet-accounts/database"
+	"github.com/sirupsen/logrus"
+	"gitlab.com/NebulousLabs/errors"
+)
+
+// TestParseConfiguration ensures that we properly parse and validate all
+// required environment variables.
+func TestParseConfiguration(t *testing.T) {
+	// Fetch current state of env and make sure we restore it on exit.
+	{
```

Comment:
could've been a separate helper function called`restoreEnv` - I have this in a blocker PR if you want to copy paste

Reply:
https://github.com/SkynetLabs/skynet-accounts/issues/138